plasma_framework/contracts/src/framework/utils/ExitPriority.sol

```diff
@@ -3,21 +3,32 @@ pragma solidity ^0.5.0;
 import "../../utils/TxPosLib.sol";
 
 library ExitPriority {
+
     /**
-     * @dev formula of priority is as folllowed: (exitableAt || txPos || nonce).
-     * The first 64 bit for exitableAt, following 128 bits of txPos and then 64 bits of nonce.
-     * The combinition of 'exitableAt' and 'txPos' is the priority for Plasma M(ore)VP protocol.
-     * 'exitableAt' only provide granuality of block, thus add 'txPos' to provide priority of transaction.
+     * @dev Given a utxo position and a unique ID, returns an exit priority.
+     * The combination of 'exitableAt' and 'txPos' is the priority for Plasma M(ore)VP protocol.
+     * 'exitableAt' only provide granularity of block, thus add 'txPos' to provide priority of transaction.
+     * @param exitId Unique exit identifier.
+     * @return An exit priority.
+     *   Anatomy of returned value, most significant bits first
+     *   42 bits - timestamp (exitable_at); unix timestamp fits into 32 bits
```

Comment:
> unix timestamp fits into 32 bits

Since 32 bits timestamp has the 2038 issue, I think it might be better to comment the similar way like we do in `txPos` on how long can this schema hold.

Reply:
I will fix the comments :+1: